src/stacks.js

```diff
@@ -0,0 +1,21 @@
+import { showBlockstackConnect } from '@stacks/connect';
+import { AppConfig, UserSession } from '@stacks/auth';
+
+const appConfig = new AppConfig();
+const userSession = new UserSession({ appConfig });
+
+function authenticate(sendToSignIn) {
```

Comment:
We tend to follow inline export style

```suggestion
export const userSession = new UserSession({ appConfig });

export function authenticate(sendToSignIn) {
```

(and remove export at bottom)

Reply:
Thanks, good tip 👍